### DISCUSSION TITLE
My favicons aren't being used

### DISCUSSION THREAD

**Discussion by amrllkmn** [nunocoracao/blowfish#655] [Q&A] [answered]
Hi, I've been building my personal website with the theme, and I followed the documentation to use my own favicons (put the files in the `static/` of my project folder). However when I run `hugo server`, it still shows the default blowfish logo. What am I missing?

**Comment by amrllkmn** [comment, marked as answer]
Apologies, it seems that after I deployed a second time after the initial push with favicon folders did it update.

**Comment by nunocoracao** [comment]
@amrllkmn also cache :)

**Reply by jamesbraza** [reply]
I hit this too, and my issue also was resolved with a hard refresh of Chrome (to clear the cache)
